app/controllers/help_controller.rb

```diff
@@ -0,0 +1,10 @@
+class HelpController < ApplicationController
+  before_action :users_only, only: [:first_login]
+  layout false
```

Comment:
This is out of scope so we don't need to do anything about it now, but I had a thought: could `layout` be set to `false` _only_ when these pages are accessed through the modal? It would look so much nicer if direct links to the help text (which we sometimes use in news posts or ticket replies, and which are what users get if JavaScript is disabled) showed them in the full site style.

Reply:
Sweet -- thank you! You have no idea how excited I am about this. :D 

(I did talk to Docs a while ago about making sure the help modals start with `h2` heading levels, so it will look even better when that's done.)